### DISCUSSION TITLE
use same object twice on scene

### DISCUSSION THREAD

**Discussion by Trusiak** [enable3d/enable3d#388] [Help / Questions]
if i try to add same object on scene twice i got error:` THREE.Object3D.add: object not an instance of THREE.Object3D.`
I read that it's because three.js doesn't deep copy objects - so if i want to put the same object on the next scene, i would have to load new scene from scratch. 

Does enable3d have any hidden deep copying functionality for objects so that if I have multiple scenes I can use the same object in each scene?

**Comment by yandeu** [comment]
Is your object a model you load with the preloader?

**Reply by Trusiak** [reply]
im loading model on scene like this
`this.load.gltf(`path_to_model`)`

**Comment by Trusiak** [comment]
Ok, I created just a clone object - but I had to use the SkeletonUtils library from THREE.js because simple clone doesn't work well for me.
